<ui version="4.0" >
 <class>ConfigForm</class>
 <widget class="QDialog" name="ConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="labelTitle" >
     <property name="text" >
      <string>Configure Rockbox Utility</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="buttonOk" >
     <property name="text" >
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="buttonCancel" >
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTabWidget" name="tabConfiguration" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProxy" >
      <attribute name="title" >
       <string>&amp;Proxy</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QRadioButton" name="radioNoProxy" >
         <property name="text" >
          <string>&amp;No Proxy</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="radioSystemProxy" >
         <property name="text" >
          <string>Use &amp;System values</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QRadioButton" name="radioManualProxy" >
         <property name="text" >
          <string>&amp;Manual Proxy settings</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Proxy Values</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Host:</string>
            </property>
            <property name="buddy" >
             <cstring>proxyHost</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="proxyHost" >
            <property name="frame" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&amp;Port:</string>
            </property>
            <property name="buddy" >
             <cstring>proxyPort</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="proxyPort" />
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="proxyUser" />
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="proxyPass" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&amp;Username</string>
            </property>
            <property name="buddy" >
             <cstring>proxyUser</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>P&amp;assword</string>
            </property>
            <property name="buddy" >
             <cstring>proxyPass</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLanguage" >
      <attribute name="title" >
       <string>&amp;Language</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="listLanguages" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScrobbler" >
      <attribute name="title" >
       <string>&amp;Scrobbler</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
